src/migrations/20210304141005-add-announce-field-to-application.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+exports.up = function(db, cb) {
+    db.runSql(
+        `
+    ALTER TABLE client_applications ADD COLUMN announced boolean DEFAULT false;
```

Comment:
does this end up announcing all existing applications in our DB?

Reply:
> The new column is initially filled with whatever default value is given (null if you don't specify a DEFAULT clause).

https://www.postgresql.org/docs/9.5/ddl-alter.html

Reply:
could we just add a: 

`UPDATE client_applications set announced=true`

Reply:
I was afraid of locking. So the query that looks for unannounced applications also filters out rows with the column set to `null`. That should avoid announcing all apps. 
I just know from earlier projects that updating all rows in a migration can be a bit risky, but we might not be speaking millions of rows here, so maybe premature worrying.

Reply:
After checking the doc, I see we've already set all rows to 'f', I've added the extra
`Update client_applications set announced=true` query